sdk/monitor/azure-monitor-opentelemetry-exporter/tests/test_base_exporter.py

```diff
@@ -642,6 +643,23 @@ def test_transmission_empty(self):
         status = self._base._transmit([])
         self.assertEqual(status, ExportResult.SUCCESS)
 
```

Comment:
Perhaps add a test for the constructor of the exporter and passing in a dummy credential

Reply:
Done